cli/module_loader.rs

```diff
@@ -748,11 +748,13 @@ impl NpmModuleLoader {
       .read_to_string(&file_path)
       .map_err(AnyError::from)
       .with_context(|| {
-        let mut msg = "Unable to load ".to_string();
+        let mut msg =
+          "Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import ".to_string();
```

Comment:
This is being done even if it's not a directory. I'll push a change with some fixes.